post-processor/amazon-import/post-processor.go

```diff
@@ -63,6 +64,10 @@ func (p *PostProcessor) Configure(raws ...interface{}) error {
 		p.config.S3Key = "packer-import-{{timestamp}}.ova"
 	}
 
+	if p.config.LicenseType == "" {
+		p.config.LicenseType = "BYOL"
```

Comment:
Funny enough. When I came back to my local build after it got farther in the import it received and error saying AWS licensing isn't valid for the 'Linux' platform.

```
1495058104,,ui,say,==> file: Running post-processor: manifest
1495058104,,ui,error,Build 'file' errored: 1 error(s) occurred:\n\n* Post-processor failed: Import task import-ami-fg3rhnba failed: ClientError: The chosen license type (AWS) is incompatible with the chosen platform type (Linux).
``` 

That's what I get for not testing with the CLI first.